src/rvsdg/schema.egg

```diff
@@ -48,6 +49,97 @@
 (function Gamma (Operand VecOperand VecVecOperand) Body) ;; branching
 (function Theta (Operand VecOperand VecOperand) Body) ;; loop
 
+;; Substitution
+
+;;                   e  [ x -> v ]
+(function SubstExpr (Expr i64 Operand) Expr)
```

Comment:
I don't see why this shouldn't be
`(function SubstExpr (Expr Operand Operand) Expr)`
This would allow you to substitute operands for other operands, not just arguments.

Reply:
Some initial rambling, but let's talk about this in person:

I think if we allow subst target arbitrary e-classes, rather than concrete args, then these rules become less "pure computation" and it becomes harder to "delete subst once we know the target doesn't occur," since something might get unioned in the future.

Reply:
Okay, semantics too hard lets just do args for now